CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.17.0 - 2023-06-29
```

Comment:
nit: Perhaps use "unreleased" so we don't forget to update the header when doing the release.

```suggestion
# unreleased
```